src/components/index.js

```diff
@@ -11,4 +11,4 @@ export { default as SidebarHeader } from './SidebarHeader';
 export { default as Tooltip } from './Tooltip';
 export { default as Input } from './Input';
 export { default as ProgressBar} from './ProgressBar';
-export { default as SocialButton} from './SocialButton';
\ No newline at end of file
+export { default as SocialButton} from './SocialButton';export { default as Loader } from './Loader';
```

Comment:
Might be a bit cleaner on a separate line 😄.

Reply:
yes haha , thanks fede